web/js/components/layer/settings/imagery-search.js

```diff
@@ -46,8 +46,17 @@ export default function ImagerySearch({ layer }) {
       }
       return maxExtent[i];
     });
+    const xDiff = Math.abs(extent[0] - extent[2]);
```

Comment:
Since we are using this block (lines 49-57) again at lines 79-87 I'd like to see this put into a function.

Reply:
Lines 42-48 are also reused and had already existed there before my changes, which is why I did it this way. I can refactor it and bring all of that out into a function though

Reply:
I agree, lets simplify things where possible. Thanks!